src/foolscap/test/test_appserver.py

```diff
@@ -135,15 +135,15 @@ def test_create(self):
         serverdir = os.path.join(basedir, "fl")
         d = self.run_cli("create", "--location", "localhost:1234", serverdir)
         def _check((rc,out,err)):
-            self.failUnlessEqual(rc, 0)
-            self.failUnless(os.path.isdir(serverdir))
+            self.assertEqual(rc, 0)
+            self.assertTrue(os.path.isdir(serverdir))
             # check that the directory is group/world-inaccessible, even on
             # windows where those concepts are pretty fuzzy. Do this by
             # making sure the mode doesn't change when we chmod it again.
             mode1 = os.stat(serverdir).st_mode
-            os.chmod(serverdir, 0700)
+            os.chmod(serverdir, 0o700)
```

Comment:
An extra change. :)

Reply:
This happened cuz these changes were part of the review changes made after hab's feedback on #55 . I copied those files from a stash to create #57 . Since this is how it should be for Py2/3 , should I revert it ?